decidim-proposals/app/commands/decidim/proposals/admin/update_proposal.rb

```diff
@@ -58,9 +59,17 @@ def update_proposal
             scope: form.scope,
             address: form.address,
             latitude: form.latitude,
-            longitude: form.longitude
+            longitude: form.longitude,
+            created_in_meeting: form.created_in_meeting
           )
         end
+
+        def update_proposal_author
+          proposal.coauthorships.clear
```

Comment:
Because Admin can edit the proposal. If he edit the proposal and change the author to another meeting, or decide that it is no longer a proposal that originated in a meeting, it must clean the authors, and assign again to prevent that proposal having two authors the meeting and the organization.

the same is being done here: https://github.com/decidim/decidim/blob/fe3b6cc2241cf5b9699e604035af81a05988eeb5/decidim-proposals/app/commands/decidim/proposals/update_proposal.rb#L63

Reply:
What if the proposal had multiple authors before editing? Aren't we losing information?

Reply:
If the admin updates a proposal, it can only be official or originated in a meeting.
Therefore, the author will be the organization or a meeting. It should not be the case that you have more than one author in this kind of proposals

Reply:
Ok!